.github/workflows/ci-cd.yml

```diff
@@ -153,3 +192,98 @@ jobs:
           use_qemu: 'true'
           registry_user: ${{ secrets.ECOSYSTEM_SERVICE_USER_DOCKER_HUB }}
           registry_token: ${{ secrets.ECOSYSTEM_ACCESS_TOKEN_DOCKER_HUB }}
+
+  create_prerelease:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    needs: build_and_push
```

Comment:
for clarity you should also have a check here that it is not a PR 
'if: (github.event_name != 'pull_request')'